asset-transfer-ledger-queries/chaincode-go/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/hyperledger/fabric-samples/chaincode/marbles02/go
+
+go 1.12
```

Comment:
can bump this to 1.14

Reply:
🛠️ Ah I copied this over from marbles02.